_pages/en_US/hiyacfw-setup.md

```diff
@@ -0,0 +1,131 @@
+1. Download the latest python file of [Just-A-DFC](https://github.com/yourkalamity/just-a-dfc/releases)
+1. Extract the hiyaCFW Helper archive anywhere on your PC
+1. Launch two terminal windows
+  - On MacOS, you can launch a terminal window by entering spotlight search (`Command` + `Space Key`) & typing "Terminal"
+  - On Linux, launch a terminal instance via your application launcher
+1. On the first terminal, `cd` to where you extracted the hiyaCFW Helper and type `py3 HiyaCFW_Helper.py`
+1. On the second terminal, `cd` to where you downloaded `just-a-DFC.py` and type `py3 just-a-DFC.py`
+{% endcapture %}
+
+<div id="memoryPitInstructions" class="blanktabcontent">{{ memoryPitInstructions | markdownify }}</div>
+<div id="flipnoteLennyInstructions" class="blanktabcontent">{{ flipnoteLennyInstructions | markdownify }}</div>
+
+<script>
+	let tabcontent = document.getElementsByClassName("blanktabcontent");
+	let tablinks = document.getElementsByClassName("tablinks");
+
+	function openTab(evt, tabName) {
+		let element;
+
+		for (element of tabcontent) {
+			element.style.display = "none";
+		}
+
+		for (element of tablinks) {
+			element.className = element.className.replace("btn--primary", "btn--info");
+			if (!element.className.includes('btn--info'))
+				element.className += " btn--info";
+		}
+
+		document.getElementById(tabName).style.display = "block";
+		evt.currentTarget.className = evt.currentTarget.className.replace("btn--info", "btn--primary");
+	}
+
+	// Get the element with id="defaultOpen" and click on it
+	document.getElementById("defaultOpen").click();
+</script>
+
+## Section II - Adding hiyaCFW files to your SD card
+![](https://camo.githubusercontent.com/538f1d70409d6b38170ef7d845b7818bd902fd87/68747470733a2f2f696d6167652e6962622e636f2f68687a4b524c2f53637265656e2d53686f742d323031382d31302d31382d61742d31362d33302d31382e706e67)
+
+1. Switch over to the hiyaCFW window
+1. Click the `...` button in the `NAND file with No$GBA footer` box
+1. Navigate to your NAND backup, and click `Open`
+1. Press `Start`
+1. In the new pop-up window, navigate to the root of your SD card, and press `OK`.
+   - The process may take several minutes
+1. When the application says "Done", close out of hiyaCFW Helper
+1. Close the terminal window
+
+## Section III - Maximizing available space
+
+This is the section you should redo if you get a screen saying "An Error Has Occured"
+{: .notice--info}
+
+The Nintendo DSi System Menu has a limit on how much free space it could have on the storage device hosting the menu. While this doesn't occur on the actual NAND (since the chip would never exceed 128 MB), it can happen using an SD card. The factor of what is allowed and what isn't goes by every other range of two gigabytes. For example, while having 0-2 GB worth of free space would allow your system menu to boot, 2-4 GB doesn't. The same goes for 4-6 GB vs 6-8, up until you get to your SD card size.
```

Comment:
I usually say Nintendo DSi because that's the official product name.

Reply:
I think imo it depends on how its being used, "Nintendo DSi Menu" is the actual name, but "your DSi" I would leave the nintendo out as its commonly referred to as just the "DSi" and especially when it shows up several times in a paragraph it can start to feel really excessive imo.

As for this case I would say "Nintendo DSi Menu" but "DSi System Menu" as the "System Menu" is not a name Nintendo ever uses